packages/reactstrap-validation-select/AvSelect.js

```diff
@@ -237,6 +291,9 @@ class AvSelect extends AvBaseInput {
             };
           },
         }}
+        options={!attributes.loadOptions && [...options, ...newOptions]}
```

Comment:
if `attributes.loadOptions` is truthy, a boolean value gets passed as the `options` prop, causing a prop type warning:

```
Warning: Failed prop type: Invalid prop `options` of type `boolean` supplied to `AsyncPaginateBase`, expected an array.
    in AsyncPaginateBase (created by AsyncPaginate)
    in AsyncPaginate (created by AvSelect)
    in AvSelect (created by AvSelectField)
    in div (created by FormGroup)
    in FormGroup (created by AvGroup)
    in AvGroup (created by AvSelectField)
    in AvSelectField (created by AvResourceSelect)
    in AvResourceSelect
    in form (created by Form)
    in Form (created by AvForm)
    in AvForm (created by AvFormResult)
    in AvFormResult
    in div (created by StoryPreview)
    in StoryPreview (created by ReadmeContent)
    in div (created by ReadmeContent)
    in ReadmeContent
```

Perhaps this should be:
```jsx
options={!attributes.loadOptions ? [...options, ...newOptions] : []}
```

Reply:
Yeah probably should do `|| undefined` since if we do pass in an options array it will take precedence over `loadOptions`.